src/lib/index.js

```diff
@@ -379,7 +379,9 @@ lib.getPlotDiv = function(el) {
 
 lib.isPlotDiv = function(el) {
     var el3 = d3.select(el);
-    return el3.size() && el3.classed('js-plotly-plot');
+    return el3.node() instanceof HTMLElement &&
```

Comment:
if someone calls `Plotly.Plots.supplyDefaults({ data: [], layout: {}});` which should be allowed.